server/routes/status.ts

```diff
@@ -184,33 +186,34 @@ GROUP BY CONCAT("OffchainThreads".chain, "OffchainThreads".community);
   });
 
   // get invites for user
-  const invites = await models.InviteCode.findAll({
-    where: {
-      invited_email: user.email,
-      used: false,
-    },
-  });
+  // const invites = await models.InviteCode.findAll({
```

Comment:
So -- are invites deprecated now as well?

Reply:
correction: keeping these. We're getting rid of `InviteLinks` instead